circus/tests/test_client.py

```diff
@@ -78,41 +77,51 @@ def test_client(self):
         yield self.stop_arbiter()
 
 
+TMP_FILE = './HOOK_CALLED'
+
+
 def long_hook(*args, **kw):
-    time.sleep(5)
+    with open(TMP_FILE, 'a'):
```

Comment:
Could you use a file created via `tempfile` instead? I don't really like creating files in the current directory for the tests, and it's not very side-effect proof.